test-appimagetool.sh

```diff
@@ -90,4 +92,6 @@ hash2=$(sha256sum appimagetool.AppImage.2 | awk '{print $1}')
 if [ "$hash1" != "$hash2" ]; then
     echo "Hash $hash1 doesn't match hash $hash2!"
     exit 1
+else
+    echo "Hashes of *.AppImage.1 and *.AppImage.2 match"    # Explicit succcess message in Travis log is better than silence
```

Comment:
Perhaps post the hash value, like in the line above? Now that we have reproducible builds, the hash should always stay the same.

Reply:
Ok.